files/install-deps.yaml

```diff
@@ -17,11 +17,6 @@
           # httpd & deps
           - python3-mod_wsgi
           - mod_ssl
-          - python3-pip # not included in base fedora:31 image, needed for next task
-          # temporary dependencies
-          - krb5-devel
-          - gcc
-          - python3-devel
```

Comment:
I can remember that the first time I tried to install packit through its setup.py I had to install krb and python devel packages. But I don't remember which package was using them for building C bindings. I think they are not needed here because packit is installed through pip but I am wondering how can we build packit from source if with this packages it is failing now?

Reply:
> packit is installed through pip

actually, it's installed (52fb387b7) from copr (i.e. with `dnf`)

> how can we build packit from source if with this packages it is failing now?

It was failing only in Github workflow, it still works OK locally and in Zuul

Reply:
> I can remember that the first time I tried to install packit through its setup.py I had to install krb and python devel packages. But I don't remember which package was using them for building C bindings. I think they are not needed here because packit is installed through pip but I am wondering how can we build packit from source if with this packages it is failing now?

`krb5-devel` is needed by `gssapi`, and now also `rpkg` (indirectly) requires `curl-devel` (and both require `gcc` and `python3-devel` of course), but as Jirka said, packit is now installed from copr.